records/2023-Q3-activity-report.md

```diff
@@ -10,13 +10,14 @@ Adrien Piquerez, 80%;
 Jamie Thompson, 100%;
 Sébastien Doeraene, 100%;
 Guillaume Martres, 100% until August 31, 20% since;
+VirtusLab team: Jędrzej Rochala, 100%
```

Comment:
This line needs to stay at the end. Otherwise the "VirtusLab team" header displays as applying to all following staff members.